ui/desktop/app/routes/application.js

```diff
@@ -34,9 +33,13 @@ export default class ApplicationRoute extends Route {
   async beforeModel() {
     this.intl.setLocale(['en-us']);
     await this.session.setup();
-    const theme = this.session.get('data.theme');
-    this.toggleTheme(theme);
     await this.clusterUrl.updateClusterUrl();
+    const theme = this.session.get('data.theme');
+    /* eslint-disable-next-line ember/no-controller-access-in-routes */
+    const controller = this.controllerFor(
+      'scopes.scope.projects.settings.index',
+    );
+    controller.toggleTheme(theme);
```

Comment:
I had to do this so the color selection is applied when the app refreshes, on refresh we take the user to the landing page. If I were to set this in `settings` route `beforemodel` hook, then after a refresh, the user would only see their selection when they navigate to the settings route which didn't seem like a good UX.  Let me know your thoughts on this approach

Reply:
on second thought, maybe we shouldn't take them to the landing page(targets) on refresh? 🤔 By refresh I mean, a forced manual refresh (cmd+ r)